addons/html_builder/static/src/plugins/badge_option.js

```diff
@@ -0,0 +1,16 @@
+import { Plugin } from "@html_editor/plugin";
+import { withSequence } from "@html_editor/utils/resource";
+import { registry } from "@web/core/registry";
+
+class BadgeOptionPlugin extends Plugin {
+    static id = "BadgeOption";
+    resources = {
+        builder_options: [
+            withSequence(10, {
```

Comment:
I don't know about the sequence to be honest, I just copy/pasted from `WidthOptionPlugin`. Is there a way I can determine the right sequence right now, or do I simply remove the sequence until we have more options implemented and we can finalize the order?